src/generator.rs

```diff
@@ -104,6 +104,7 @@ mod tests {
         "transport_protocol": 17
       },
       {
+        "sni": "",
```

Comment:
If Option, let's check the field isn't serialized as null if None. Let's use the #[serde_with::skip_serializing_none].

Reply:
ok